### PR TITLE
[Chore] Update to Swift 2.3

### PR DESCRIPTION
**Pull request by quentinR**

### CODE REVIEW

**Comment by htinlinn** [comment]
It's odd. Somehow, Xcode 8 on my computer doesn't recognize that the project is already on Swift 2.3 and forces me to go through the migration steps again. But, it comes out with no change.

**Comment by htinlinn** [comment]
👍

File: Example/Gemfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'cocoapods', '1.1.0.rc.3'
```
Comment (MichaelCampbell) [inline Example/Gemfile]: cocoa pods 1.1.1 is available

File: Example/Makefile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+RUBY_VERSION=$(shell cat .ruby-version | cut -d- -f2)
```
Comment (MichaelCampbell) [inline Example/Makefile]: you can remove `| cut -d- -f2`

- [ ] remove `| cut -d- -f2`

File: Example/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+![Yoshi](../Images/Yoshi_logo.jpg)
```
Comment (MichaelCampbell) [inline Example/README.md]: @htinlinn do you think this image should be in this example project README?

Comment (htinlinn) [inline Example/README.md]: No. Let's reserve the image for the main README. We didn't use the banner for the best practices document in our style guide so let's keep it consistent for now.

Comment (quentinR) [inline Example/README.md]: Fixed

**Pull request merged**